.github/workflows/wheels.yml

```diff
@@ -109,52 +122,48 @@ jobs:
         with:
           fetch-depth: 0
 
-      # TODO: Build wheels from sdist again
-      # There's some sort of weird race condition?
-      # within Github that makes the sdist be missing files
-
-      # We need to build wheels from the sdist since the sdist
-      # removes unnecessary files from the release
-      - name: Download sdist (not macOS)
-        #if: ${{ matrix.buildplat[1] != 'macosx_*' }}
+      - name: Download sdist and pandas_tests
         uses: actions/download-artifact@v4
         with:
-          name: sdist
+          name: sdist-and-tests
           path: ./dist
 
-      - name: Output sdist name (macOS)
-        id: save-path
-        shell: bash -el {0}
-        run: echo "sdist_name=$(ls ./dist)" >> "$GITHUB_ENV"
-
-      # Python version used to build sdist doesn't matter
-      # wheel will be built from sdist with the correct version
+      # The sdist will be corrupted by cibuildwheel/macOS sometimes somehow

```

Comment:
What is the corruption that happens?

Reply:
I just moved an old comment. This was a pre-existing bug (it's just not apparent from the diff).


Here's the original comment
https://github.com/pandas-dev/pandas/pull/53007/files#diff-75627befeafda60a526c116b32f6fdef9bf57bcf456f840d7592c1d6f13facb4L112-L114